packages/app/src/hooks/useWindowsHotfix.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect } from 'react';
+import { type MenuIds, useRunMenuCommand } from './useMenuCommands';
+
+interface HotfixWindow extends Window {
+    __tauri_hotfix?: boolean;
+}
+declare let window: HotfixWindow;
+
+const isWindowsPlatform = typeof navigator !== 'undefined' && navigator.userAgent.includes('Win64');
+
+if(isWindowsPlatform) {
+    console.warn('Hotfix applied for Windows platform');
+}
+
+/**
+ * Applies a keyboard shortcut hotfix for Windows platform.
+ */
+export const useWindowsHotfix = () => {
+    const runMenuCommandImpl = useRunMenuCommand();
+
+    // @see https://github.com/Ironclad/rivet/issues/261
+    useEffect(() => {
+        if (typeof window === 'undefined' || !isWindowsPlatform || window.__tauri_hotfix) {
```

Comment:
A hotfix is more like "fixing something that's currently running (i.e. hot)", that's why I was saying "hotkey fix"

Reply:
agreed, not a great name. lol